core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -177,6 +182,21 @@ var ContributorDashboardAdminPage = function() {
       'Submit Question Right Element taking too long to appear');
     expect(await contributionRights.getText()).toBe('Allowed');
   };
+
+  this.checkstatslistelements = async function(number) {
```

Comment:
The `number` here creates confusion with the type `number` in the type script. Maybe use elementsCount*